website/app/Http/Controllers/EventsController.php

```diff
@@ -40,6 +41,8 @@ public function index()
    */
   public function store(CreateEventRequest $request)
   {
+    @include status.blade.php
```

Comment:
You should never use include inside a function block, the reason is that the include happens every time this function gets called instead of just once globally.